amethyst_rendy/src/submodules/gather.rs

```diff
@@ -54,7 +54,13 @@ impl CameraGatherer {
             convert::<_, Vector3<f32>>(transform.global_matrix().column(3).xyz()).into_pod();
 
         let proj: [[f32; 4]; 4] = (*camera.as_matrix()).into();
-        let view: [[f32; 4]; 4] = convert::<_, Matrix4<f32>>(transform.view_matrix()).into();
+        let view: [[f32; 4]; 4] = convert::<_, Matrix4<f32>>(
+            transform
+                .global_matrix()
+                .try_inverse()
+                .expect("Unable to get inverse of camera transform"),
```

Comment:
Can we avoid the `expect` here? I'd really prefer falling back to the identity matrix and logging a (one-time) error.

Reply:
This has been updated and removed with a solution from @termhn